### DISCUSSION TITLE
[Feature Request]: Adjust configuration files manually

### DISCUSSION THREAD

**Discussion by Tklaversma** [beyondcode/herd-community#491] [Ideas]
It would be nice if we're able to adjust the `php.ini`, `nginx.conf`, `redis.conf`, etc.

Herd already has the most used settings in the GUI, but being able to adjust these configurations give the people more control over their services.

**Comment by Tklaversma** [comment]
@sschlein you got any update regarding this feature request?

**Comment by mathiaswillburger** [comment]
Agree. I adjusted the configs manually, but as I was new to Herd, it took quite a while to figure out where all the files were located in the config folders (especially in the Windows version).

**Comment by Tklaversma** [comment]
@sschlein Any update on this?

**Comment by sschlein** [comment]
What are you looking for? The files are available in your Herd directory :)

**Reply by Tklaversma** [reply]
@sschlein I don't have Herd installed, was just checking for open FR's. Guess we can close this one then.
